src/visit/traversal.rs

```diff
@@ -344,6 +344,23 @@ where
         topo
     }
 
+    /// Create a new `Topo` with initial nodes.
+    ///
+    /// If any incoming edges to the initial nodes will be ignored.
+    pub fn with_initials<G, I>(graph: G, i: I) -> Self
+    where
+        G: IntoNeighborsDirected + Visitable<NodeId = N, Map = VM>,
+        I: IntoIterator<Item = N>,
+    {
+        Topo {
+            tovisit: i
```

Comment:
Slight rename to make the API clearer.
```suggestion
    pub fn with_initials<G, I>(graph: G, initials: I) -> Self
    where
        G: IntoNeighborsDirected + Visitable<NodeId = N, Map = VM>,
        I: IntoIterator<Item = N>,
    {
        Topo {
            tovisit: initials
```